mmdeploy/codebase/mmcls/models/backbones/shufflenet_v2.py

```diff
@@ -10,6 +10,9 @@
 @FUNCTION_REWRITER.register_rewriter(
     func_name='mmcls.models.backbones.shufflenet_v2.InvertedResidual.forward',
     backend='ncnn')
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmcls.models.backbones.shufflenet_v2.InvertedResidual.forward',
+    backend='torchscript')
```

Comment:
may use `backend=Backend.TORCHSCRIPT.vaule`, it would be easy to change the backend name of if necessary later.